src/Framework/Analyzer/src/MinimalActions/DiagnosticDescriptors.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.AspNetCore.Analyzers.MinimalActions
+{
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("MicrosoftCodeAnalysisReleaseTracking", "RS2008:Enable analyzer release tracking")]
+    internal static class DiagnosticDescriptors
+    {
+        internal static readonly DiagnosticDescriptor DoNotUseModelBindingAttributesOnMinimalActionParameters = new(
+            "ASP0003",
+            "Do not use model binding attributes with Map actions",
+            "{0} should not be specified for a {1} delegate parameter",
```

Comment:
> Can we point them at the correct solution?

Could you recommend what would be?

Reply:
/cc @BrennanConroy

Reply:
I don't know, I haven't used map actions much

Reply:
There's a few things model binding can do - say that you want to bind a loose collection of key value pairs from route / query / form (no support), say you want to bind a subset, or say that you want to bind some value with a different name. It's not trivial to figure out what the user's intent is, to give people something very prescriptive.